<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>DeploymentView</class>
    <widget class="QWidget" name="DeploymentView">
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QGroupBox" name="groupBox">
                    <property name="title">
                        <string>Compiler Settings</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout">
                        <item row="0" column="0">
                            <widget class="QLabel" name="label">
                                <property name="text">
                                    <string>Input Model:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="1">
                            <widget class="QLineEdit" name="lineEditInput"/>
                        </item>
                        <item row="0" column="2">
                            <widget class="QPushButton" name="btnBrowseInput">
                                <property name="text">
                                    <string>...</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="0">
                            <widget class="QLabel" name="label_2">
                                <property name="text">
                                    <string>Output Engine:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="1">
                            <widget class="QLineEdit" name="lineEditOutput"/>
                        </item>
                        <item row="1" column="2">
                            <widget class="QPushButton" name="btnBrowseOutput">
                                <property name="text">
                                    <string>...</string>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="0">
                            <widget class="QLabel" name="label_3">
                                <property name="text">
                                    <string>Target Platform:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="1">
                            <widget class="QComboBox" name="cmbTarget"/>
                        </item>
                        <item row="3" column="0">
                            <widget class="QLabel" name="label_4">
                                <property name="text">
                                    <string>Precision:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="3" column="1">
                            <widget class="QComboBox" name="cmbPrecision">
                                <item>
                                    <property name="text">
                                        <string>FP16</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>INT8</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>FP32</string>
                                    </property>
                                </item>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QPushButton" name="btnCompile">
                    <property name="minimumHeight">
                        <number>40</number>
                    </property>
                    <property name="text">
                        <string>Compile Model</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QProgressBar" name="progressBar">
                    <property name="value">
                        <number>0</number>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QPlainTextEdit" name="txtLog">
                    <property name="readOnly">
                        <bool>true</bool>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>